roiextractors/extractors/hdf5imagingextractor/hdf5imagingextractor.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
+from pathlib import Path
+from roiextractors import ImagingExtractor
+from roiextractors.extraction_tools import get_video_shape
+
+try:
+    import h5py
+    HAVE_H5 =True
+except ImportError:
+    HAVE_H5 = False
+
+
+class Hdf5ImagingExtractor(ImagingExtractor):
+    extractor_name = 'Hdf5Imaging'
+    installed = HAVE_H5  # check at class level if installed or not
+    is_writable = True
+    mode = 'file'
+    installation_mesg = "To use the Hdf5 Extractor run:\n\n pip install h5py\n\n"  # error message when not installed
+
+    def __init__(self, file_path, mov_field='mov', sampling_frequency=None,
+                 channel_names=None):
+        assert HAVE_H5, self.installation_mesg
+        ImagingExtractor.__init__(self)
+        self.filepath = Path(file_path)
+        self._sampling_frequency = sampling_frequency
+        self._mov_field = mov_field
+        assert self.filepath.suffix in ['.h5', '.hdf5'], ""
+        self._channel_names = channel_names
+
+        with h5py.File(file_path, "r") as f:
+            if 'mov' in f.keys():
+                self._video = f[self._mov_field]
+                self._sampling_frequency = self._video.attrs["fr"]
+                self._start_time = self._video.attrs["start_time"]
+                self.metadata = self._video.attrs["meta_data"]
```

Comment:
I think these attributes should be input args to init, to try to handle different kinds of HDF5 inputs